pages.de/common/kubectl-delete.md

```diff
@@ -0,0 +1,32 @@
+# kubectl delete
+
+> Löschen von Kubernetes-Ressourcen.
+> Mehr Informationen: <https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#delete>.
+
+- Löschen eines bestimmten Pods:
+
+`kubectl delete pod {{pod_name}}`
+
+- Löschen eines bestimmten Deployments:
+
+`kubectl delete deployment {{deployment_name}}`
+
+- Löschen eines bestimmten Nodes:
+
+`kubectl delete node {{node_name}}`
+
+- Löschen aller Pods in einem bestimmten Namespaces:
+
+`kubectl delete pods --all --namespace {{namespace}}`
+
+- Löschen aller Deployments und Services in einem bestimmten Namespace:
+
+`kubectl delete deployments,services --all --namespace {{namespace}}`
+
+- Alle Nodes löschen:
+
+`kubectl delete nodes --all`
+
+- Löschen von Resourcen, welche in einer YAML Datei definiert sind :
+
+`kubectl delete --filename {{path/to/manifest.yaml}}`
```

Comment:
```suggestion
`kubectl delete --filename {{pfad/zu/manifest}}.yaml`
```
I think we can move `.yaml` outside to better align with other command pages. I will create a new PR to make this change in original page.